### PR TITLE
client lib updates for 3.14

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: automation/requirements.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
 cryptography<3.4
-pulpcore==3.14.0
+pulpcore==3.14.1
```
Comment (ekohl) [inline automation/requirements.txt:2]: I don't see this change in this PR, but looks like it reflects https://github.com/theforeman/pulpcore-packaging/commit/bc731b26d2823f9aade576e31e0dcbfb8fbc5eac

File: automation/requirements.txt [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 pulp-file==1.8.1
 pulp-container==2.7.0
 pulp-deb==2.13.0
-pulp-rpm==3.13.2
+pulp-rpm==3.13.3
```
Comment (ekohl) [inline automation/requirements.txt:6]: Looks like this reflects https://github.com/theforeman/pulpcore-packaging/commit/eb2a83e5734b05a240d700aedd11aabcd9d04964.

**Review by ekohl — APPROVED** [review]
And I intended to approve ...

File: automation/requirements.txt (hunk 2 above)
Comment (evgeni) [reply automation/requirements.txt:6]: Correct

File: automation/requirements.txt (hunk 1 above)
Comment (evgeni) [reply automation/requirements.txt:2]: Correct

**Pull request merged**